services/blockchain-connector/shared/sdk/index.js

```diff
@@ -13,12 +13,111 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const { refreshNetworkStatus } = require('./network');
+const { Signals } = require('lisk-service-framework');
+
+const {
+	getLastBlock,
+	getBlockByID,
+	getBlocksByIDs,
+	getBlockByHeight,
+	getBlocksByHeightBetween,
+} = require('./blocks');
+
+const {
+	getTransactionByID,
+	getTransactionsByIDs,
+	getTransactionsFromPool,
+	postTransaction,
+	dryRunTransaction,
+} = require('./transactions');
+
+const {
+	getGenesisHeight,
+	getGenesisBlockID,
+	getGenesisBlock,
+	getGenesisConfig,
+} = require('./genesisBlock');
+
+const {
+	getGenerators,
+	getForgingStatus,
+	updateForgingStatus,
+	invokeEndpointProxy,
+	getSchemas,
+	getRegisteredActions,
+	getRegisteredEvents,
+	getRegisteredModules,
+	getNodeInfo,
+	getSystemMetadata,
+	validateBLSKey,
+} = require('./endpoints');
+
+const {
+	getPeers,
+	getConnectedPeers,
+	getDisconnectedPeers,
+	getPeersStatistics,
+} = require('./peers');
+
+const { getEventsByHeight } = require('./events');
+const { refreshNetworkStatus, getNetworkStatus } = require('./network');
+const { setSchemas, setMetadata } = require('./schema');
 
 const init = async () => {
 	await refreshNetworkStatus();
+
+	// Cache all the schemas
+	setSchemas(await getSchemas());
+	setMetadata(await getSystemMetadata());
+
+	// Download the genesis block, if applicable
+	await getGenesisBlock();
+
+	// Register listener to update the nodeInfo cache only when it updates
+	const updateNodeInfoCacheListener = getNodeInfo.bind(null, true);
```

Comment:
Just curious if it could be written like below as well?

```suggestion
	const updateNodeInfoCacheListener = () => getNodeInfo(true);
```

Reply:
yeah, asked for self clarification.